tests/test_impersonated_credentials.py

```diff
@@ -391,6 +417,28 @@ def test_signer_email(self):
         credentials = self.make_credentials(target_principal=self.TARGET_PRINCIPAL)
         assert credentials.signer_email == self.TARGET_PRINCIPAL
 
+    def test_sign_endpoint(self):
```

Comment:
this does not test if sign_bytes method is really using the correct url.

Reply:
it was not before ) but added